app/src/androidTest/java/ch/epfl/sdp/TestTools.java

```diff
@@ -83,54 +81,40 @@ static void sleep(int milliseconds) {
     static void sleep() {
         sleep(2000);
     }
+
     /**
      * Rounds a double to 5 digits after the comma
-     * @param coor
+     *
+     * @param
      * @return
      */
-    static double roundCoordinate(double coor){
-        return (double)Math.round(coor * 100000d) / 100000d;//fast rounding to 5 digits
-    }
 
-    /**
-     * Rounds a location to 5 digits after the comma
-     * @param l
-     * @return
-     */
-    static Location roundLocation(Location l){
-        if(l == null){
-            throw new IllegalArgumentException("Location can't be null");
-        }
-        double latitude = l.getLatitude();
-        double longitude = l.getLongitude();
-        latitude = roundCoordinate(latitude);
-        longitude = roundCoordinate(longitude);
-        l.setLatitude(latitude);
-        l.setLongitude(longitude);
-        return l;
-    }
 
-    static Location newLoc(double lati,double longi){
-        Location res =  new Location("provider");
+    static Location newLoc(double lati, double longi) {
+        Location res = new Location("provider");
         res.reset();
         res.setLatitude(lati);
         res.setLongitude(longi);
         return res;
     }
-    static boolean expandedLocEquals(Location loc1, Location loc2){
+
+    static boolean expandedLocEquals(Location loc1, Location loc2) {
```

Comment:
I don't believe so, since to compare Locations we still need to explicitly check only latitude and longitude (Location.equals() also check for many other things like altitude, timestamp,...)
This function is used in ConcretePositionAggregatorTest.
I will change its name though: equalLatLong

Reply:
Nice